playbooks/group_vars/db.yml

```diff
@@ -1,27 +1,27 @@
 ---
 # mirsg.infrastructure.postgresql - setting up the database
 postgresql_database:
-  database_name: "{{ xnat_db.postgres_xnat_database }}"
```

Comment:
`postgres_xnat_database`, `postgres_xnat_user` and `postgres_xnat_password` have been changed to match what was set for OMERO - `postgresql_database` etc. I don't think the `xnat` part of the name is needed.